Source/Core/DolphinQt/Config/Mapping/MappingIndicator.cpp

```diff
@@ -577,15 +592,24 @@ CalibrationWidget::CalibrationWidget(ControllerEmu::ReshapableInput& input,
 void CalibrationWidget::SetupActions()
 {
   const auto calibrate_action = new QAction(tr("Calibrate"), this);
+  const auto center_action = new QAction(tr("Center and Calibrate"));
```

Comment:
The `addAction(QAction*)` overload does not take ownership so this will leak memory unless you set the parent manually with `this` like those other lines.

Reply:
Fixed in latest revision (force-pushed so I don't have to squash later)